index.html

```diff
@@ -30,12 +30,12 @@
             <div class="link-image">
                 <p class="connect-link">LET’S CONNECT</p>
             </div>
-                <ul class="connect-icon">
+                <div class="connect-icon">
                     <a href="#" class="fa fa-twitter"></a>
                     <a href="#" class="fa fa-linkedin"></a>
-                    <a href="#" class="fa fa-angellist"></i></a>
-                    <a href="#" class="fa fa-medium"></i></a>
-                </ul>        
+                    <a href="#" class="fa fa-angellist"></a>
+                    <a href="#" class="fa fa-medium"></a>
+                </div>        

```

Comment:
- [ ] [REQUIRED] The list of icons  uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)